python/ray/_private/runtime_env/pip.py

```diff
@@ -361,3 +366,5 @@ def modify_context(
                 "installing the runtime_env `pip` packages."
             )
         context.py_executable = virtualenv_python
+        virtualenv_activate = _PathHelper.get_virtualenv_activate(target_dir)
+        context.command_prefix += ["source %s 1>&2" % (virtualenv_activate)]
```

Comment:
Should we also support windows?

Reply:
If you know how to do it, we can try in this PR, but as you mentioned offline, a prerequisite is for `pip` to be enabled on Windows, which is being worked on here https://github.com/ray-project/ray/pull/22449.  We won't be able to test it without `pip`, so perhaps we should just merge this PR first.  I can follow up with @mattip about this later.

Reply:
OK, I don't have windows development machine. Let's me add a TODO and merge this PR first.

Reply:
Sorry, could you put the TODO in `get_virtualenv_activate` since that is what needs to change?

Reply:
OK, but `source` command also don't supported in Windows?

Reply:
Redefine to `get_virtualenv_activate_command`.